tests/python_tests/test_preemption.py

```diff
@@ -25,90 +37,233 @@ def get_beam_search_seq_len_300() -> GenerationConfig:
     generation_config.num_return_sequences = generation_config.num_beams
     return generation_config
 
-scheduler_params_list = [({"num_kv_blocks": 2, "block_size": 32, "dynamic_split_fuse": True, "max_num_batched_tokens": 256, "max_num_seqs": 256}, get_greedy()),
-                         ({"num_kv_blocks": 2, "block_size": 32, "dynamic_split_fuse": False, "max_num_batched_tokens": 256, "max_num_seqs": 256}, get_greedy()),
-                         ({"num_kv_blocks": 10, "block_size": 32, "dynamic_split_fuse": True}, get_greedy_seq_len_300()),
-                         ({"num_kv_blocks": 10, "block_size": 32, "dynamic_split_fuse": False}, get_greedy_seq_len_300()),
-                         ({"num_kv_blocks": 34, "block_size": 32, "dynamic_split_fuse": True, "max_num_batched_tokens": 256, "max_num_seqs": 256}, get_beam_search()),
-                         ({"num_kv_blocks": 34, "block_size": 32, "dynamic_split_fuse": False, "max_num_batched_tokens": 256, "max_num_seqs": 256}, get_beam_search()),
-                         ({"num_kv_blocks": 100, "block_size": 32, "dynamic_split_fuse": True}, get_beam_search_seq_len_300()),
-                         ({"num_kv_blocks": 100, "block_size": 32, "dynamic_split_fuse": False}, get_beam_search_seq_len_300())]
+
+scheduler_params_list = [
+    (
+        {
+            "num_kv_blocks": 2,
+            "block_size": 32,
+            "dynamic_split_fuse": True,
+            "max_num_batched_tokens": 256,
+            "max_num_seqs": 256,
+        },
+        get_greedy(),
+    ),
+    (
+        {
+            "num_kv_blocks": 2,
+            "block_size": 32,
+            "dynamic_split_fuse": False,
+            "max_num_batched_tokens": 256,
+            "max_num_seqs": 256,
+        },
+        get_greedy(),
+    ),
+    (
+        {"num_kv_blocks": 10, "block_size": 32, "dynamic_split_fuse": True},
+        get_greedy_seq_len_300(),
+    ),
+    (
+        {"num_kv_blocks": 10, "block_size": 32, "dynamic_split_fuse": False},
+        get_greedy_seq_len_300(),
+    ),
+    (
+        {
+            "num_kv_blocks": 34,
+            "block_size": 32,
+            "dynamic_split_fuse": True,
+            "max_num_batched_tokens": 256,
+            "max_num_seqs": 256,
+        },
+        get_beam_search(),
+    ),
+    (
+        {
+            "num_kv_blocks": 34,
+            "block_size": 32,
+            "dynamic_split_fuse": False,
+            "max_num_batched_tokens": 256,
+            "max_num_seqs": 256,
+        },
+        get_beam_search(),
+    ),
+    (
+        {"num_kv_blocks": 100, "block_size": 32, "dynamic_split_fuse": True},
+        get_beam_search_seq_len_300(),
+    ),
+    (
+        {"num_kv_blocks": 100, "block_size": 32, "dynamic_split_fuse": False},
+        get_beam_search_seq_len_300(),
+    ),
+]
```

Comment:
It's hard to extract the actual change. If you want to update the code style, you should do it in a separate PR. It seems that the whole PR is more dedicated to code style rather then tests enablement, so you can keep the change.